### PR TITLE
[17.0][FIX] partner_firstname: Wrong contact type

### PR DESCRIPTION
**Pull request by AEstLo**
There is a typo in the views of contacts

### CODE REVIEW

File: partner_firstname/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 {
     "name": "Partner first name and last name",
     "summary": "Split first name and last name for non company partners",
-    "version": "17.0.1.0.3",
+    "version": "17.0.1.0.4",
```
Comment (rousseldenis) [inline partner_firstname/__manifest__.py]: Don't update this. It will be by ocabot

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: partner_firstname/__manifest__.py (hunk 1 above)
Comment (AEstLo) [reply partner_firstname/__manifest__.py]: Thanks! I thought that I had to update it so odoo.sh knows it has to update this module to apply the changes :)
I'm reverting that

Comment (AEstLo) [reply partner_firstname/__manifest__.py]: done

**Review by rousseldenis — APPROVED** [review]

**Review by houssine78 — APPROVED** [review]

**Comment by houssine78** [comment]
Salut @rousseldenis I think it can be merged ;)

**Review by leemannd — APPROVED** [review]

**Comment by houssine78** [comment]
@pedrobaeza this PR is ready for merging :)

**Pull request merged**